### PR TITLE
Add image_uri column in aws_ecr_image table

### PR DESCRIPTION
**Pull request by bigdatasourav**
# Integration test logs
<details>
  <summary>Logs</summary>

```
NA
```
</details>

# Example query results
<details>
  <summary>Results</summary>

```
> select
  artifact_name,
  artifact_type,
  metadata,
  results
from
  trivy_scan_artifact as a,
  aws_ecr_image as i
where
  artifact_name = image_uri
  and repository_name = 'hello';
+-----------------------------------------------------------------------------------------------------------------------------+-----------------+------------------------------------------
| artifact_name                                                                                                               | artifact_type   | metadata                                 
+-----------------------------------------------------------------------------------------------------------------------------+-----------------+------------------------------------------
| 097353333455.dkr.ecr.ap-south-1.amazonaws.com/hello@sha256:984804a8ed358e0a3bce2df193c6b4d687076cc3a8f9606d5e30f3660e8f6e60 | container_image | {"DiffIDs":["sha256:cc4590d6a7187ce8879dd
|                                                                                                                             |                 | ab7a3accfc7bc55624676e"],"ImageConfig":{"
|                                                                                                                             |                 | pkg-divert --local --rename --add /sbin/i
|                                                                                                                             |                 | u0026\u0026 echo 'Apt::AutoRemove::Sugges
|                                                                                                                             |                 | ginx.conf"},{"author":"Karthik Gaekwad","
|                                                                                                                             |                 | 7T08:30:32.6982242Z","created_by":"/bin/s
|                                                                                                                             |                 | 77e13f88fdb70f85ef35c4f89a6e0953e499b1687
|                                                                                                                             |                 |                                          

```
</details>

### CODE REVIEW

File: aws/table_aws_ecr_image.go [hunk 1]
```diff
@@ -157,3 +165,25 @@ func listAwsEcrImages(ctx context.Context, d *plugin.QueryData, h *plugin.Hydrat
...
+	// AWS follows the below image URI format -
+	// with tag - {aws_account_id}.dkr.ecr.{region}.amazonaws.com/{repository name}:{first tag}
+	// without tag - {aws_account_id}.dkr.ecr.{region}.amazonaws.com/{repository name}@image_digest
+	if len(image.ImageTags) == 0 {
+		uri := commonColumnData.AccountId + ".dkr.ecr." + commonColumnData.Region + ".amazonaws.com/" + *image.RepositoryName + "@" + *image.ImageDigest
+
+		return uri, nil
+	}
+
+	uri := commonColumnData.AccountId + ".dkr.ecr." + commonColumnData.Region + ".amazonaws.com/" + *image.RepositoryName + ":" + image.ImageTags[0]
```
Comment (cbruno10) [inline aws/table_aws_ecr_image.go:186]: @bigdatasourav Does AWS always take the first tag to form the URI? What happens if you use a different tag when building the URI and then pass that URI to other tools like Trivy?

File: aws/table_aws_ecr_image.go [hunk 2]
```diff
@@ -44,6 +45,13 @@ func tableAwsEcrImage(_ context.Context) *plugin.Table {
 				Description: "The sha256 digest of the image manifest.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "image_uri",
+				Description: "It is a unique identifier that specifies the location and version of the image.",
```
Comment (cbruno10) [inline aws/table_aws_ecr_image.go:50]: Can you please look at the description of the repository_uri column and follow a similar style?

Comment (bigdatasourav) [reply aws/table_aws_ecr_image.go:50]: updated.

File: aws/table_aws_ecr_image.go (hunk 1 above)
Comment (bigdatasourav) [reply aws/table_aws_ecr_image.go:186]: Yes, AWS always take the first tag to form the URI. Trivy can scan image URIs with other valid tags.

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**